dss-api.yml

```diff
@@ -1259,6 +1271,25 @@ paths:
           description: Token to manage retries.  End users constructing queries should not set this parameter.
           required: false
           type: string
+        - name: per_page
+          in: query
+          description: Max number of results to return per page. When using `output_format raw` the `per_page` size is
+            limit to no more than 10 to avoid excessively large response sizes.
+          required: false
+          type: integer
+          format: int32
+          minimum: 10
+          maximum: 500
+          default: 500
+        - name: start_at
+          in: query
+          description: >
+            **Search-After-Context**.
```

Comment:
What's the significance of this line?

Reply:
Copy-paste from /POST search spec. Removed.